components/build/src/lib.rs

```diff
@@ -228,9 +238,12 @@ impl<'a> ProtobufGenerator<'a> {
         }
     }
 
-    /// Directory containing input protobuf files.
+    /// Directory containing input protobuf files. This directory is also
+    /// added to includes because in most cases proto files in input directory depends on
```

Comment:
There are no `includes` for user, so this must be altered. Documentation of `add_path` must also be expanded. It's not obvious why can't I call `with_input_dir` multiple times for every my directory.